sdk/monitor/monitor-query/swagger/metric-namespaces.md

```diff
@@ -22,5 +22,5 @@ disable-async-iterators: true
 api-version-parameter: choice
 hide-clients: true
 v3: true
-use-core-v2: false
+use-core-v2: true
```

Comment:
NIT: consider removing this line all-together and also remove the following bit that specifies the code gen version so you always use the latest one available with most recent bug fixes:
```
use-extension:
  "@autorest/typescript": "6.0.0-beta.8"
```

Consider doing this in all 4 swagger configs.

Reply:
Done